roles/openshift_logging/tasks/install_logging.yaml

```diff
@@ -32,6 +32,17 @@
   with_items: "{{object_def_files.files | map(attribute='path') | list | sort}}"
   changed_when: no
 
+- oc_obj:
+    state: absent
+    name: "{{item.name}}"
+    kind: "{{item.kind}}"
+    kubeconfig: "{{ mktemp.stdout }}/admin.kubeconfig"
+    namespace: "{{openshift_logging_namespace}}"
+  with_items:
+  - {name: logging-kibana, kind: route}
+  - {name: logging-kibana-ops, kind: route}
+  - {name: kibana-proxy, kind: oauthclient}
```

Comment:
Can we use the `oc_route` module specifically for the routes? I'm not sure if it would handle this scenario for us though... it would probably just create a new one and leave the old one dangling